### PR TITLE
fix: fixes for testcontainers

### PR DESCRIPTION
**Pull request by rupansh-gob**

### CODE REVIEW

File: ssr/src/state/canisters.rs [hunk 1]
```diff
@@ -218,17 +214,19 @@ impl<const A: bool> Canisters<A> {
...
+            // // TODO: this is temporary
+            // let blacklisted =
+            //     HashSet::from([Principal::from_text("rimrc-piaaa-aaaao-aaljq-cai").unwrap()]);
+            // let orchestrator = self.orchestrator().await;
+            // Ok(orchestrator
+            //     .get_all_available_subnet_orchestrators()
+            //     .await?
+            //     .into_iter()
+            //     .filter(|subnet| !blacklisted.contains(subnet))
+            //     .collect())
```
Comment (ravibazz) [inline ssr/src/state/canisters.rs:229]: Do we still need this?

Comment (rupansh-gob) [reply ssr/src/state/canisters.rs:229]: Is it safe to uncomment again?

Comment (ravibazz) [reply ssr/src/state/canisters.rs:229]: no I meant can we not remove this completely?

Comment (rupansh-gob) [reply ssr/src/state/canisters.rs:229]: > no I meant can we not remove this completely?

We would need to add it back in the future no?
Or does this user index load balance across all subnets

Comment (rupansh-gob) [reply ssr/src/state/canisters.rs:229]: ah just noticed #446 lets merge this first

**Review by ravibazz — APPROVED** [review]

**Pull request merged**